frontend/packages/vkt/src/interfaces/publicReservation.ts

```diff
@@ -0,0 +1,9 @@
+import { PublicExamEvent } from 'interfaces/publicExamEvent';
+import { PublicPerson } from 'interfaces/publicPerson';
+
+export interface PublicReservation {
+  id: number;
+  expiresAt: number;
```

Comment:
Pitäisikö tämän olla `expiresAt: Dayjs`?

Reply:
Hmm, ok. Tietty riippuu miten tän konseptin (vissiin "aika siihen että varaus umpeutuu") haluaa mallintaa - palvelin joko voi palauttaa umpeutumisen kellonajan (kuten itse ajattelin) tai jäljellä olevan aikamäärän. Tässä vissiin haettu tuota jälkimmäistä? Tämäkin käy, mutta pieni nimeämismuutos voisi sitten auttaa: `expiresAt` viittaa mielestäni kellonlyömään, kun taas `expiresIn` olisi enemmän linjassa jäljellä olevan aikamäärän suhteen ("expires in 5 minutes" vs. "expires at midnight").

Reply:
Paitsi että ilmeisesti tämä on kuitenkin mallia timestamp. Eikö silloin olisi loogisempaa palauttaa bäkkäristä joku DateTime-tyylinen olio, vähän samaan tapaan kuin esim. akr:ssä auktorisointien termBeginDate ja termEndDate ovat `LocalDate`ja?

Reply:
Joo, tämä expiresAt on umpeutumisaika unix timestampinä. Voisin koittaa vaihtaa tuon ajan bäkkärissä LocalDateTime olioksi ja tsekata, miten Spring tuon response bodyna käsittelee ja saako frontendille näin loogisen aikaleiman josta mäpätä Dayjs instanssi

Reply:
Tää commit 8df0f20 vaikuttaa toimivalta ratkaisulta